src/shmem_comm.h

```diff
@@ -140,6 +146,8 @@ shmem_internal_get(void *target, const void *source, size_t len, int pe)
 {
     int node_rank;
 
+    if (len == 0) return;
```

Comment:
Should this be an assert?  I imagined 'get' being on the critical path; but now that I think of it, maybe not.